spec/import_spec.rb

```diff
@@ -44,6 +44,40 @@ def upload_file!(name, ext='csv')
     find_button('Import').click
   end
 
+  context "posts index" do
+    before do
+        Author.create!([{name: 'John', last_name: 'Doe'}, {name: 'Jane', last_name: 'Roe'}])
+        add_post_resource({
+          validate: true,
+          headers_rewrites: {
+              :'Author Name' => :author_id
+          },
+          before_batch_import: ->(importer) {
+            authors_names = importer.values_at(:author_id)
+            # replacing author name with author id
+            authors   = Author.where(name: authors_names).pluck(:name, :id)
+            #{"Jane" => 2, "John" => 1}
+            options = Hash[*authors.flatten]
+            importer.batch_replace(:author_id, options)
+          }
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.